From a4dd84f42abea1ff9cdd15e156c63776c0bf854b Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Wed, 3 Apr 2013 11:41:43 +0200
Subject: [PATCH] Remove extra <sys/mount.h> include

Added in b856e9b9e94a96ab396662db7a503280fa267dc3, it basically useless.
(And it is not a standard header, so might be missing on some OSes.)
---
 kio/kio/kautomount.cpp |    1 -
 1 file changed, 1 deletion(-)

diff --git a/kio/kio/kautomount.cpp b/kio/kio/kautomount.cpp
index e145b04..4ddf6d7 100644
--- a/kio/kio/kautomount.cpp
+++ b/kio/kio/kautomount.cpp
@@ -24,7 +24,6 @@
 #include <kdirnotify.h>
 #include <kdebug.h>
 #include <kmountpoint.h>
-#include <sys/mount.h>
 
 /***********************************************************************
  *
-- 
1.7.10.4

